splunkoncall_connector.py

```diff
@@ -401,7 +402,7 @@ def _handle_update_incident(self, param):
             "entity_display_name": param_name,
             "state_message": param_message
         }
-
+        self.debug_print("Endpoint created for _handle_create_update_incident action: {0}".format(endpoint))
```

Comment:
Do we really need all the debug messages here and in rest of the code?

Reply:
There is an update in the static test pipeline and according to that, we should have at least 2 logs statement in the methods. That's why we have made these changes in the code.